User/Library/UserPcd.c

```diff
@@ -50,4 +50,4 @@ BOOLEAN  _gPcd_FeatureFlag_PcdFatReadOnlyMode                        = _PCD_VALU
 UINT32   _gPcd_BinaryPatch_PcdSerialRegisterStride                   = 0;
 UINT8    _gPcd_FixedAtBuild_PcdUefiImageFormatSupportNonFv           = 0x00;
 UINT8    _gPcd_FixedAtBuild_PcdUefiImageFormatSupportFv              = 0x03;
```

Comment:
These PCDs are primarily used for BaseTools construction and [determine](https://github.com/acidanthera/audk/blob/master/MdePkg/Library/BaseUefiImageLib/UefiImageLib.c#L31-L44) supported formats (PE/UE). Because for them [`UEFI_IMAGE_FORMAT_SUPPORT_SOURCES=0x02`](https://github.com/acidanthera/audk/blob/master/BaseTools/Source/C/Makefiles/header.makefile#L147l), `_gPcd_FixedAtBuild_PcdUefiImageFormatSupportNonFv` will change nothing.

Reply:
Same comment as below, I'm trying to ensure that OC userspace tools are unsurprising by matching the Pcd configuration of OC itself - unless in any given case there's a good reason not to.